lib/actions/collection.js

```diff
@@ -25,10 +25,10 @@ module.exports = {
 
       if (trackKey) xhrs[trackKey] = xhr;
 
-      const promise = new Promise((resolve, reject) => {
-        xhr.done(collection => resolve(collection.map(model => model.attributes)));
-        xhr.fail(reject);
-      });
+      const promise = new Promise((resolve, reject) => xhr
```

Comment:
Totally optional: You could inline this const now and do `return new Promise(...)`